dotCMS/src/main/webapp/html/portlet/ext/dotai/dotai.js

```diff
@@ -136,11 +136,12 @@ const writeModelToDropdown = async () => {
         }
 
         const newOption = document.createElement("option");
+        console.log(JSON.stringify(dotAiState.config, null, 2));
```

Comment:
remove?

Reply:
Or maybe log less info instead? It may be useful for Support Team when troubleshooting issues.

Reply:
It was a debugguing thing. Removing it.,